docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -6,7 +6,7 @@ internal static class Program
     {
         static void Main(string[] args)
         {
-            TakeRows.Example();
+            LightGbmRanking.Example();
```

Comment:
please don't commit this change. only use it for local testing. it only creates unnecessary merge conflicts.